runtime/near-vm-logic/src/logic.rs

```diff
@@ -147,12 +152,12 @@ impl<'a> VMLogic<'a> {
             promises: vec![],
             receipt_to_account: HashMap::new(),
             total_log_length: 0,
-            current_protocol_version,
+            protocol_features,
         }
     }
 
-    /// Legacy initialization method that doesn't pass the protocol version and uses the last
-    /// protocol version before the change was introduced.
+    /// Legacy initialization method that doesn't pass the protocol protocol features,
+    /// just disabling them to stay compatible with the old behavior.
     pub fn new(
```

Comment:
Can we just remove this method now? I don't understand why we need it.

Reply:
@Longarithm I believe you made the change. Is there any reason to keep this method?

Reply:
@bowenwang1996 I don't see a reason to keep it